recipes/configs/qwen2_5/1.5B_full.yaml

```diff
@@ -0,0 +1,74 @@
+# Config for multi-device full finetuning in full_finetune_distributed.py
+# using a Qwen2.5 1.5B model
+#
+# This config assumes that you've run the following command before launching:
+#   tune download Qwen/Qwen2.5-1.5B-Instruct --ignore-patterns None
+#
+# To launch on 2 devices, run the following command from root:
+#   tune run --nproc_per_node 2 full_finetune_distributed --config qwen2_5/1.5B_full
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training:
+#   tune run --nproc_per_node 2 full_finetune_distributed --config qwen2_5/1.5B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#
+# This config works best when the model is being fine-tuned on 2+ GPUs.
+# Single device full finetuning requires more memory optimizations. It's
```

Comment:
prob same here: 1.5 * 8 = 12, so most users will not need any optimizations to run this